WcaOnRails/spec/factories/users.rb

```diff
@@ -16,15 +16,24 @@
     factory :admin do
       name "Mr. Admin"
       email "admin@worldcubeassociation.org"
-      software_team true
+      after(:create) do |user|
+        software_team = FactoryGirl.create(:team, friendly_id: 'software', name: 'Software Team', description: 'Does software')
```

Comment:
How does this work if we create two admins? Will we end up creating 2 software teams?